cloud/amazon/s3_lifecycle.py

```diff
@@ -65,10 +65,10 @@
     choices: [ 'enabled', 'disabled' ]
   storage_class:
     description:
-      - "The storage class to transition to. Currently there is only one valid value - 'glacier'."
+      - "The storage class to transition to. Currently there are supported 2 valid values - 'glacier' or 'standard_ia'."
```

Comment:
This doesn't read well. Try...

> Currently there are two supported values - 'glacier' or 'standard_ia'
 
Also you should note something about standard_ia only being available from Ansible version 2.2